src/DotNetWorker/Definition/BindingDirection.cs

```diff
@@ -3,9 +3,18 @@
 
 namespace Microsoft.Azure.Functions.Worker
 {
```

Comment:
Looks like we tackled some of the same types. The PR I mentioned earlier deals with some of these